doc/api/core/grids.rst

```diff
@@ -31,6 +31,7 @@ variable spacing in the three coordinate directions.
 
 .. jupyter-execute::
    :hide-code:
+   :stderr:
```

Comment:
Alright, I was able to bring the number of `WARNING: Cell printed to stderr:` warnings down from 18 to 12 by adding these `:stderr:` options in `jupyter-execute` blocks that (indirectly) call `extract_all_edges`. I'll see if I can find the remaining 12 cells that cause this. An alternative would be to make `vtkExtractEdges` not print to stderr, but I'm not sure that can be done? I also have no clue why this is only now causing problems...

Reply:
I think I know of another way to do this globally

Reply:
I think this diff will suppress the warnings:

```diff
diff --git a/pyvista/_vtk.py b/pyvista/_vtk.py
index 45bcb8d9..94efc2a1 100644
--- a/pyvista/_vtk.py
+++ b/pyvista/_vtk.py
@@ -112,6 +112,7 @@ if VTK9:
         vtkFloatArray,
         vtkIdList,
         vtkIdTypeArray,
+        vtkLogger,
         vtkLookupTable,
         vtkOutputWindow,
         vtkPoints,
@@ -629,7 +630,7 @@ else:  # pragma: no cover
     try:
         from vtk.vtkCommonKitPython import buffer_shared, vtkAbstractArray, vtkWeakReference
     except ImportError:
-        from vtk.vtkCommonCore import buffer_shared, vtkAbstractArray, vtkWeakReference
+        from vtk.vtkCommonCore import buffer_shared, vtkAbstractArray, vtkLogger, vtkWeakReference

     import vtk

diff --git a/pyvista/core/filters/data_set.py b/pyvista/core/filters/data_set.py
index f6def045..b1d36ee7 100644
--- a/pyvista/core/filters/data_set.py
+++ b/pyvista/core/filters/data_set.py
@@ -1404,6 +1404,8 @@ class DataSetFilters:
         See :ref:`cell_centers_example` for more examples using this filter.

         """
+        # vtkExtractEdges improperly uses INFO for debugging messages
+        _vtk.vtkLogger.SetStderrVerbosity(_vtk.vtkLogger.VERBOSITY_OFF)
         alg = _vtk.vtkExtractEdges()
         alg.SetInputDataObject(self)
         if use_all_points:
```

This is indeed an upstream issue though. `vtkExtractEdges` is not supposed to use INFO logs for debug messages like this.

I tried having `SetStderrVerbosity` in the `conf.py` but I think it needs to be called in whatever context the `jupyter-execute` is happening

Reply:
This does have the consequence of turning of logging for all other VTK classes once this is run though...

Reply:
FYI https://github.com/pyvista/pyvista/discussions/3495#discussioncomment-4170082

Reply:
And https://github.com/pyvista/pyvista/issues/3612

Reply:
Ok, I will let you implement that fix.
I will revert my previous 'fix', as this prints the warning in the produced docs 😅 
![image](https://user-images.githubusercontent.com/15089458/212483417-9568e107-37ed-433c-9e48-a051d07f54fb.png)

Reply:
I quickly applied your vtkLogger fix, let's see what we get :)

Reply:
> build succeeded.

Ah, perfect 😄 Will leave the other changes you wanted to perform to you @banesullivan.